### PR TITLE
style(console): username

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Requirements:
image

## Testing

image
image
image

### CODE REVIEW

**Comment by charIeszhao** [comment]
The second line should left align wtih the first line
image

File: packages/console/src/components/UserName/index.module.scss [hunk 1]
```diff
@@ -4,13 +4,23 @@
   font: var(--body-medium);
   color: var(--color-text);
 
-  .userId {
+  .title {
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
```
Comment (charIeszhao) [inline packages/console/src/components/UserName/index.module.scss:10]: `overflow: hidden` is included in `_.text-ellipsis` mixin

File: packages/console/src/components/UserName/index.module.scss [hunk 2]
```diff
@@ -4,13 +4,23 @@
...
+    @include _.text-ellipsis;
```
Comment (charIeszhao) [inline packages/console/src/components/UserName/index.module.scss:11]: And I think all table columns should have this `text-ellipsis` mixin applied

File: packages/console/src/components/UserName/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/UserName/index.module.scss]: updated.

File: packages/console/src/components/UserName/index.module.scss (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/UserName/index.module.scss:10]: I have tried it before and found it caused some unexpected behaviors. 
e.g.
image
Also, some data need to be displayed in more than one line.

Comment (charIeszhao) [reply packages/console/src/components/UserName/index.module.scss:10]: Then these cases are the exceptions, we can use CSS to cancel the ellipsis. What do you think?

Comment (xiaoyijun) [reply packages/console/src/components/UserName/index.module.scss:10]: I think we can not assume data in the table data should use text ellipsis by default.
1. It changes the default style of the table cell
2. Using CSS to cancel the ellipsis requires us to write a lot of styles to cover the ellipsis needs, which will make the code not so intuitive

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**